paymentsheet/src/main/java/com/stripe/android/paymentsheet/navigation/PaymentSheetScreen.kt

```diff
@@ -97,29 +99,52 @@ internal sealed interface PaymentSheetScreen {
 
         @Composable
         override fun Content(viewModel: BaseSheetViewModel, modifier: Modifier) {
-            val state by viewModel.paymentOptionsState.collectAsState()
-            val isEditing by viewModel.editing.collectAsState()
-            val isProcessing by viewModel.processing.collectAsState()
+            val state by selectSavedPaymentMethodsInteractor.state.collectAsState()
 
             SavedPaymentMethodTabLayoutUI(
```

Comment:
Is it worth adding a constructor that just takes the interactor? Then we can add some UI tests easily.

Reply:
yep! I can do that in a follow up